docs/Orchestrator.md

```diff
@@ -0,0 +1,34 @@
+### Adding transactions
+
+1) Use the truffle console to encode the function call as follows.
+
+```
+# Sync tx
+web3.eth.abi.encodeFunctionCall({
+  name: 'sync',
+  type: 'function',
+  inputs: [],
+}, []);
+
+
+# Gulp tx
+web3.eth.abi.encodeFunctionCall({
+  name: 'gulp',
+  type: 'function',
+  inputs: [{
+      type: 'address',
+      name: 'token'
+  }],
+}, ['0xD46bA6D942050d489DBd938a2C909A5d5039A161']);
+```
+
+2) Admin invokes `addTransaction` with the destination contract address and `bytes`
+as encoded from step 1.
+
+### Current list of transactions
+
+1. UniswapV2 (AMPL-ETH): sync() => `0x08a564924C26D8289503bbaA18714B9C366dF9a5`, `0xfff6cae9`
```

Comment:
This is what I get for the current transactions array:
0. UniV2 DAI-AMPL => 0x08a564924C26D8289503bbaA18714B9C366dF9a5
1. UniV2 ETH-AMPL => 0xc5be99A02C6857f9Eac67BbCE58DF5572498F40c
2. Balancer ETH-AMPL => 0xeefb11cA05c6F0d5252757Dbe35fFC4458108e89

Reply:
Is this not what we have here?